### ISSUE TITLE
[MathDrawer] How to display correction in the panel ?

### ISSUE THREAD

**Issue by ddoyen** [PremierLangage/components#4]
After the answer is validated, the board is recreated with the initial parameters, erasing the modifications made by the student (most often, point locations).

I'd like to keep the modifications made by the students after the validation.

Also, I'd like to be able to run an additional script to display a correction in the JSXGraph panel.

**Comment by mciissee** [comment]
here is an example implementing these features : https://pl.u-pem.fr/filebrowser/option?name=test_pl&path=Yggdrasil/Mathematics/exercises/complex_numbers/affixe.pl

Can you test it on multiple browser and tell me the version of the unsupported browsers ?

**Comment by ddoyen** [comment]
It doesn't work : the panel is displayed but without point. Same behavior with Firefox or Chromium.

And the other component issues (smartFence, keyboards) are back...

**Comment by ddoyen** [comment]
It seems to work now. :-)

**Comment by mciissee** [comment]
There is some delay before CDN cache is refreshed so you cannot see the modifications in realtime

**Comment by ddoyen** [comment]
Actually, the components work (as they are expected to work) only after reloading the page.

**Comment by mciissee** [comment]
we'll see all this together earlier

**Comment by ddoyen** [comment]
Ok !

**Comment by mciissee** [comment]
All your codes should work nicely on all browser now can you confirm it?

**Comment by ddoyen** [comment]
I've tried some exercises (with different components). Everything seems ok. Thanks a lot !

**Comment by mciissee** [comment]
OK for the next release I will the remove the little hack I did. You don't have to do anything on your side I will keep all your codes backward compatible.

unfortunately I cannot modify the documentation for now with the hack you will wait for the next release to see the right documentation.

important for the MathDrawer component the new property **script** cannot include any dango statement like` {% if %}` or `{{ variableintepolation }}` so if you want to provide a correction after max attempt look at the file affixe.pl to do the same thing I did.
